lib/bulk/common.js

```diff
@@ -563,6 +563,23 @@ class FindOperators {
     this.s = bulkOperation.s;
   }
 
+  /**
```

Comment:
Since we're not implementing the shell helper (`bulk.find().arrayFilters`), we don't need these changes so we can remove them.